src/draw/renesas/dave2d/lv_draw_dave2d_utils.c

```diff
@@ -75,7 +75,7 @@ d2_s32 lv_draw_dave2d_cf_fb_get(void)
             d2_fb_mode = d2_mode_argb8888;
             break;
         case DISPLAY_IN_FORMAT_32BITS_RGB888: ///< RGB888,   32 bits
-            d2_fb_mode = d2_mode_rgb888;
+            d2_fb_mode = d2_mode_argb8888;
```

Comment:
I think this line deserve a comment or something like this:
```
case DISPLAY_IN_FORMAT_32BITS_ARGB888:
case DISPLAY_IN_FORMAT_32BITS_RGB888:
             d2_fb_mode = d2_mode_argb8888;
```

This way it makes explicitly that only mode supported is 4-byte colour pattern, even if the alpha channel is not used.

Reply:
Comment added